source/Octodiff.Tests/Octodiff.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net451</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We should test against both `net6.0` and a framework version `net462`. It might be as easy as adding a `net462` here and the tests will magically happen.